### PR TITLE
Az.CognitiveServices - Update SDK to 7.0, Improve error message

### PR DESCRIPTION
**Pull request by yangyuan**
## Description

Update SDK to 7.0
Improve error message when server responses empty body

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/CognitiveServices/CognitiveServices/CognitiveServicesAccount/CognitiveServicesAccountBaseCmdlet.cs [hunk 1]
```diff
@@ -84,6 +84,11 @@ protected void RunCmdLet(Action action)
             }
             catch (ErrorException ex)
             {
+                if (ex.Body == null)
```
Comment (yangyuan) [inline src/CognitiveServices/CognitiveServices/CognitiveServicesAccount/CognitiveServicesAccountBaseCmdlet.cs]: When service response empty string, Body is null. so display ex.Message instead.

Other changes are all just SDK update.

**Comment by VeryEarly** [comment]
Hi @yangyuan ,

Your PR looks good to me, could you please format your code in this convention?
Just to improve the readability.

this.CognitiveServicesClient.Accounts.Update(this.ResourceGroupName, 
                                                                         this.Name,
                                                                         new CognitiveServicesAccount()
                                                                         {
                                                                           Properties = properties
                                                                         });

File: src/CognitiveServices/CognitiveServices/CognitiveServicesAccount/AddAzureCognitiveServicesAccountNetworkRule.cs [hunk 2]
```diff
@@ -156,21 +156,26 @@ public override void ExecuteCmdlet()
                         break;
                 }
 
-                var properties = new JObject();
-                properties["networkAcls"] = JToken.FromObject(accountACL);
-                this.CognitiveServicesClient.Accounts.Update(this.ResourceGroupName, this.Name, null, null, properties);
+                var properties = new CognitiveServicesAccountProperties();
+                properties.NetworkAcls = accountACL;
+                this.CognitiveServicesClient.Accounts.Update(this.ResourceGroupName, this.Name,
+                    new CognitiveServicesAccount()
```
Comment (markcowl) [inline src/CognitiveServices/CognitiveServices/CognitiveServicesAccount/AddAzureCognitiveServicesAccountNetworkRule.cs]: Update formatting, per comments

File: src/CognitiveServices/CognitiveServices/CognitiveServicesAccount/RemoveAzureCognitiveServicesAccountNetworkRule.cs [hunk 3]
```diff
@@ -152,21 +152,26 @@ public override void ExecuteCmdlet()
                 }
 
 
-                var properties = new JObject();
-                properties["networkAcls"] = JToken.FromObject(accountACL);
-                this.CognitiveServicesClient.Accounts.Update(this.ResourceGroupName, this.Name, null, null, properties);
+                var properties = new CognitiveServicesAccountProperties();
+                properties.NetworkAcls = accountACL;
+                this.CognitiveServicesClient.Accounts.Update(this.ResourceGroupName, this.Name,
+                    new CognitiveServicesAccount()
```
Comment (markcowl) [inline src/CognitiveServices/CognitiveServices/CognitiveServicesAccount/RemoveAzureCognitiveServicesAccountNetworkRule.cs]: update formatting, per comments

File: src/CognitiveServices/CognitiveServices/CognitiveServicesAccount/SetAzureCognitiveServicesAccount.cs [hunk 4]
```diff
@@ -152,9 +152,13 @@ public override void ExecuteCmdlet()
                     var updatedAccount = this.CognitiveServicesClient.Accounts.Update(
                         this.ResourceGroupName,
                         this.Name,
-                        sku,
-                        tags,
-                        properties);
+                        new CognitiveServicesAccount()
+                        {
+                        Sku = sku,
```
Comment (markcowl) [inline src/CognitiveServices/CognitiveServices/CognitiveServicesAccount/SetAzureCognitiveServicesAccount.cs]: indent properties

File: src/CognitiveServices/CognitiveServices/CognitiveServicesAccount/UpdateAzureCognitiveServicesAccountNetworkRuleSet.cs [hunk 5]
```diff
@@ -138,13 +138,18 @@ public override void ExecuteCmdlet()
                     psNetworkRule.DefaultAction = defaultAction.Value;
                 }
 
-                var properties = new JObject();
-                properties["networkAcls"] = JToken.FromObject(psNetworkRule.ToNetworkRuleSet());
-                this.CognitiveServicesClient.Accounts.Update(this.ResourceGroupName, this.Name, null, null, properties);
+                var properties = new CognitiveServicesAccountProperties();
+                properties.NetworkAcls = psNetworkRule.ToNetworkRuleSet();
+                this.CognitiveServicesClient.Accounts.Update(this.ResourceGroupName, this.Name,
+                    new CognitiveServicesAccount()
```
Comment (markcowl) [inline src/CognitiveServices/CognitiveServices/CognitiveServicesAccount/UpdateAzureCognitiveServicesAccountNetworkRuleSet.cs]: update formatting

**Comment by yangyuan** [comment]
@markcowl update the format as suggested :)

**Comment by VeryEarly** [comment]
> @markcowl update the format as suggested :)

Hi @yangyuan ,

github display my suggested format change in a different way :(

sent you the right one through teams, also please fix the conflicting files

**Pull request merged**